Doc/whatsnew/3.8.rst

```diff
@@ -1233,6 +1370,16 @@ them in the generated tree.
 (Contributed by Stefan Behnel in :issue:`36676` and :issue:`36673`.)
 
 
+xmlrpc
+------
+
+:class:`xmlrpc.client.ServerProxy` now supports an optional *headers* keyword
```

Comment:
```suggestion
:class:`~xmlrpc.client.ServerProxy` now supports an optional *headers* keyword
```

Is there a general "rule of thumb" that applies to the usage of inline link shortening with `~`? I typically try to use it when the inline link goes beyond the `module.function/method` or `module.class` format. I think that it might be useful here, as it would fall under the format of `module.subsection.class`; which might be a bit excessively long.

Reply:
I prefer it as is.

Reply:
@rhettinger That's fine if you prefer it as is. Could you explain when you prefer to use it or not use it? This would help me get a general idea for writing my own news entries.